hack/templates/values/values.template

```diff
@@ -226,6 +226,7 @@ ofedDriver:
       # podSelector: "app=myapp"
       # specify the length of time in seconds to wait before giving up for workload to finish, zero means infinite
       # timeoutSeconds: 300
+  useOcpDriverToolkit: true
```

Comment:
do we really want to have this defined in values ?

Reply:
Removed from helm